event_reporter/reporter/application_event_reporter.go

```diff
@@ -143,6 +160,12 @@ func (s *applicationEventReporter) StreamApplicationEvents(
 		return err
 	}
 
+	var applicationVersions *apiclient.ApplicationVersions = nil
+	liveManifests, err, _ := s.getLiveManifests(ctx, a, logCtx)
```

Comment:
can we remove third returned value from function?
it looks like same can be done with `getDesiredState`